devops/linux/docker/conf/haproxy/haproxy.cfg

```diff
@@ -36,32 +36,30 @@ frontend fe_stats-http
     bind *:30460
     use_backend     be_stats-http       if { path -i /my-stats }
  
-# frontend fe_keycloak
-#     bind *:8080
-#     use_backend     be_keycloak
 
-frontend  fe_api
```

Comment:
I feel we should leave the un-authenticated API running on 50000 through haproxy and run the authenticated API on another port. This will allow for easy development and if someone wants an production deployment they should alter the haproxy config. The Webapp should be configred to use the new port of the authenticated API.

Reply:
Hi @rcrichton . Yes, did think of that. One of my main worries was that this might introduce a security risk, as you can easily update the api port in the webapp (i.e via browser developer tools) giving you access to the unauthenticated api. Yes, one could update the haproxy config before deploying it to production, but the risk of forgetting to do that (which I think would happen), as opposed to a developer having to update their own haproxy config, if need be, seemed greater to me.

The other reason was that only the webapp uses authenticated api. The other services connect to the `backend_network` in the docker stack which uses the unautheticated api on the same port - hence saw no need to have the different ports.

Open to it though if you still think that's better

Reply:
Sure, that sounds valid to me. My main concern is that other devs like William might want to still be able to use that port for development work or other scripts or tools that might expect that port to be available. We can leave the un-autenticated API commented out in the haproxy config, but maybe we should still use a different port for the authenticated API to not conflict with what others might already be using.

Reply:
Hi @rcrichton . Okay, sure 👍🏽 Will update.

Reply:
Okay, so do you want to disable keycloak by default?  p.s both api's use a shared library, so there is not difference there from a development perspective. api-kc just makes that you have to login first before when using the webapp (as you would in prod), of which there is already a dev user pre-loaded you can use.